internal/workload/podman/podman.go

```diff
@@ -289,7 +289,7 @@ func (p *podman) GenerateSystemdService(podName string, monitoringInterval uint)
 		return nil, err
 	}
 
-	svc, err := service.NewSystemd(podName, report.Units)
+	svc, err := service.NewSystemd(podName, "pod-", report.Units)
```

Comment:
What about making it configurable?

Reply:
The prefix? I think it is dictated by Podman. @machacekondra - is that right?

Reply:
Correct:
```bash
$ podman generate systemd --help | grep pod-prefix
      --pod-prefix string         Systemd unit name prefix for pods (default "pod")

```